lua/orgmode/agenda/views/agenda.lua

```diff
@@ -37,6 +57,12 @@ local function sort_agenda_items(agenda_items)
       return a.headline:get_priority_sort_value() > b.headline:get_priority_sort_value()
     end
 
+    -- if both are on the same day check category
```

Comment:
@kristijanhusak can you help with this part - basically I want to give preference to overdue items over categories, but it doesn't seem to do the trick ...

Reply:
Ok - figured it out - it's `headline_date:is_same()` rather than `date:is_same()`